docs/create-blocks/create-blocks.md

```diff
@@ -90,6 +92,51 @@ export const ButtonBlock: WpBlockType = {
 
 > We've added a custom property `postTypes` for the blocks' settings to define on which WP Post Type it can be available.<br /> 📚 [Learn more on how we've handled blocks whitelisting in here.](../features/blocks-whitelisting.md)
 
+## 🏷️ Typing `typings.d.ts`
+
+When you create a new block, a TypeScript typing file  is automatically generated. This file provides a basic structure for your block's types, but it's important to note that you should modify and adapt these types according to the specific needs of your block.
+
+The generated typing file typically includes:
+- An Attributes interface defining the block's attributes.
+- A Props interface extending the Attributes interface and potentially including additional props.  
```

Comment:
@kuuak Should we maybe specify a bit that "block's attributes" is for the WP block, and the props are for the frontend ?

Reply:
Good point, I did specify a bit more